tests/library/permissions.spec.ts

```diff
@@ -48,6 +48,14 @@ it.describe('permissions', () => {
     expect(await getPermission(page, 'geolocation')).toBe('granted');
   });
 
+  it('should grant window-management permission when origin is listed', async ({ page, context, server, browserName }) => {
+    it.fail(browserName === 'firefox');
```

Comment:
It will actually throw in WebKit too, but the test is passing as the tests are [skipped](https://github.com/microsoft/playwright/blob/e836a863cf15569b3fede9fbf4aa54d4d57b2b8c/tests/library/permissions.spec.ts#L25) in bulk (erroneously) in WebKit. I'll update the expectations separately.

Reply:
no problem thank you